.gitignore

```diff
@@ -2143,3 +2143,6 @@ lib/libs2d_engine.a
 
 # .orig files
 *.orig
+
+# :Zone_Identifier files
+**/*Zone.Identifier
```

Comment:
```suggestion
*Zone.Identifier
```
i think this would work

Reply:
I think so too but I copied it from some other guy lol

Okay then